config/locales/en.yml

```diff
@@ -679,7 +683,8 @@ en:
         unknown_placeholder: Please explain
       disabled_reminders: >-
         (Advanced) Disable inactivity reminder (we recommend you
-        leave this unchecked)
+        leave this unchecked; note that project entries
+        must be edited to change whether or not reminders are sent)
```

Comment:
I am wondering if we should place a link to edit here, with an anchor accordingly.

Reply:
We can't do that currently, because there is no way to jump *to* a "project edit" page when users aren't logged in.  Once we fix that, that might make sense.

Reply:
Good point.